receiver/windowsperfcountersreceiver/internal/third_party/telegraf/win_perf_counters/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2019 InfluxData Inc.
```

Comment:
2020?

Reply:
This was the LICENSE file brought over from Telegraf. I don't think I should change it should I?

Reply:
I believe it was updated. https://github.com/influxdata/telegraf/blob/master/LICENSE#L3.